pkg/cmd/backuprestore/restoreutils.go

```diff
@@ -14,6 +15,17 @@ var static_pod_list = []string{
 	"kube-scheduler-pod.yaml",
 }
 
+var exists = struct{}{}
+
+var staticPodContainers = map[string]struct{}{
+	"etcd":                    exists,
+	"etcdctl":                 exists,
+	"etcd-metrics":            exists,
+	"kube-controller-manager": exists,
+	"kube-apiserver":          exists,
+	"kube-scheduler":          exists,
+}
+
```

Comment:
For readability/ergonomics, you could replace the map and `exists` token entirely with `sets.String` (from the `k8s.io/apimachinery/pkg/util/sets` package)

Reply:
I didn't know we have sets. I was emulating the `set` functionality using maps. Thanks.